.github/workflows/prerelease_staged_publish.yml

```diff
@@ -0,0 +1,54 @@
+name: Staged pre-release publish for Linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to pre-release'
+        required: true
+      schema:
+        description: 'Schema to be released'
+        required: true
+        options:
+          - fb-sles
+
+env:
+  TAG: ${{ github.event.release.tag_name }}
+  GPG_MAIL: ${{ secrets.LOGGING_GPG_MAIL }}
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
+  AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
+  AWS_REGION: "us-east-1"
+
+jobs:
+  publishing-to-s3-linux:
+    name: Publish linux artifacts into s3 staging bucket
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Publish ${{ github.event.inputs.schema }} to S3 action
+        uses: newrelic/infrastructure-publish-action@v1.2.1
+        env:
+          AWS_ACCESS_KEY_ID: ${{ env.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ env.AWS_SECRET_ACCESS_KEY }}
```

Comment:
[⚠️] I believe these should be under `secrets`, so they are not revealed while compiling.

Reply:
👌🏻